File: app/src/main/java/com/hfad/joke/Java/MainActivity.java
package com.hfad.joke.Java;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import androidx.appcompat.app.AppCompatActivity;

import com.hfad.joke.R;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    public void onClick(View view) {
        Intent intent = new Intent(this, DelayedMessageService.class);
        intent.putExtra(DelayedMessageService.EXTRA_MESSAGE, getResources().getString(R.string.response));
        startService(intent);
    }
}

File: app/src/main/java/com/hfad/joke/Kotlin/MainActivity.kt
package com.hfad.joke.Kotlin

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import com.hfad.joke.R

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun onClick(view: View) {
        val intent = Intent(this@MainActivity, DelayedMessageService::class.java)
        intent.putExtra(DelayedMessageService().EXTRA_MESSAGE, resources.getString(R.string.response))
        startService(intent)
    }
}
